modules/NuGetPackageVerifier/msbuild/VerifyPackages.cs

```diff
@@ -57,6 +59,12 @@ public override bool Execute()
                 ArtifactDirectory,
             };
 
+            if (!string.IsNullOrEmpty(SignRequestManifest) && File.Exists(SignRequestManifest))
+            {
+                arguments.Add("--sign-request");
+                arguments.Add(SignRequestManifest);
+            }
```

Comment:
If someone were to mistype the path for `SignRequestManifest` then it would be silently ignored. If they supply a value, but it doesn't exist I would have expected it to Error out or at least warn.

Reply:
Updated to make this an error.